jane/doc/parallelism-tutorial.md

```diff
@@ -133,27 +288,17 @@ end
 All we have to do to sum over the prices in a `Thing.t Tree.t` is change the
 `Tree.Leaf` case:
 
-```ocaml
-  let average_things_par (par : Parallel.t) tree =
-    let rec total par tree =
-      match tree with
-      | Tree.Leaf x -> ~total:(Thing.price x), ~count:1 (* <== new code *)
-      | Tree.Node (l, r) ->
-        let (~total:total_l, ~count:count_l), (~total:total_r, ~count:count_r) =
-          Parallel.fork_join2
-            par
-            (fun par -> total par l)
-            (fun par -> total par r)
-        in
-        ~total:(total_l +. total_r), ~count:(count_l + count_r)
-    in
-    let ~total, ~count = total par tree in
-    total /. (count |> Float.of_int)
-  ;;
+```diff
+  let rec total par tree : total:float * count:int =
```

Comment:
Did you mean the change the name of the function too?

Reply:
I've changed the references to keep using the old name.